npm/private/npm_translate_lock_state.bzl

```diff
@@ -102,12 +102,14 @@ def _validate_attrs(attr, is_windows):
         fail("only one of npm_package_lock or yarn_lock may be set")
 
 ################################################################################
-def _init_common_labels(rctx, label_store):
+def _init_common_labels(priv, rctx, label_store):
     attr = rctx.attr
 
     # data files
-    for i, d in enumerate(attr.data):
-        label_store.add("data_{}".format(i), d)
+    # only initialize if update_pnpm_lock is set since data files are unused otherwise
```

Comment:
Is there any guarantee or at least docs somewhere that ensure the data files are only used if `update_pnpm_lock`?

Can this `if` be moved and merge into the other one (within `if attr.pnpm_lock`)? I guess that just makes the "data is for lockfile" stricter and clearer...

Reply:
Yeah the [docs for `data`](https://docs.aspect.build/rules/aspect_rules_js/docs/npm_translate_lock/#data) say "Only needed when `update_pnpm_lock` is True".

Reply:
I think this branch is intentionally not under `if attr.pnpm_lock` for the pnpm lock bootstrap case.